core/thread/src/TImplicitMT.cxx

```diff
@@ -20,15 +20,13 @@
 
 #include "TError.h"
 #include "TThread.h"
-
+#include "ROOT/TPoolManager.hxx"
 #include <atomic>
-#include "tbb/task_scheduler_init.h"
-
 
-static tbb::task_scheduler_init &GetScheduler()
+static std::shared_ptr<ROOT::TPoolManager> &GetPoolManagerMT()
 {
-   static tbb::task_scheduler_init scheduler(tbb::task_scheduler_init::deferred);
-   return scheduler;
+   static std::shared_ptr<ROOT::TPoolManager> schedMT;
```

Comment:
I thought the idea was that the pool manager could 'disappear' if the using tools were all out of scope.
I.e. I would expect the static to be a weak_ptr and the function to have a conditional (re)creation of the manager.